app/javascript/admin_companies.vue

```diff
@@ -0,0 +1,165 @@
+          td.admin-table__item-value.is-text-align-center
+            ul.is-inline-buttons
+              li
+                a.a-button.is-sm.is-secondary.is-icon(
+                  :href='`/admin/companies/${company.id}/edit`'
+                )
+                  i.fas.fa-pen
+              li
+                a.a-button.is-sm.is-danger.is-icon.js-delete(
+                  @click='destroy(company)'
+                )
+                  i.fas.fa-trash-alt
+    nav.pagination(v-if='totalPages > 1')
+      pager(v-bind='pagerProps')
+</template>
+
+<script>
+import Pager from './pager'
+import toast from './toast'
+
+export default {
+  components: {
+    pager: Pager
+  },
+  mixins: [toast],
+  data() {
+    return {
+      companies: [],
+      totalPages: 0,
+      currentPage: this.getCurrentPage(),
+      loaded: false
+    }
+  },
+  computed: {
+    url() {
+      return `/api/admin/companies?page=${this.currentPage}`
+    },
+    pagerProps() {
+      return {
+        initialPageNumber: this.currentPage,
+        pageCount: this.totalPages,
+        pageRange: 5,
+        clickHandle: this.paginateClickCallback
+      }
+    }
+  },
+  created() {
+    window.onpopstate = () => {
+      this.currentPage = this.getCurrentPage()
+      this.getCompaniesPage()
+    }
+    this.getCompaniesPage()
+  },
+  methods: {
+    token() {
+      const meta = document.querySelector('meta[name="csrf-token"]')
+      return meta ? meta.getAttribute('content') : ''
+    },
+    getCompaniesPage() {
+      fetch(this.url, {
+        method: 'GET',
+        headers: {
+          'Content-Type': 'application/json; charset=utf-8',
+          'X-Requested-With': 'XMLHttpRequest',
+          'X-CSRF-Token': this.token()
+        },
+        credentials: 'same-origin',
+        redirect: 'manual'
+      })
+        .then((response) => response.json())
+        .then((json) => {
+          this.companies = json.companies
+          this.totalPages = json.total_pages
+          this.loaded = true
+        })
+        .catch((error) => {
+          console.warn('Failed to parsing', error)
+        })
+    },
+    destroy(company) {
+      if (window.confirm('本当によろしいですか？')) {
+        fetch(`/api/admin/companies/${company.id}.json`, {
+          method: 'DELETE',
+          headers: {
+            'Content-Type': 'application/json; charset=utf-8',
+            'X-Requested-With': 'XMLHttpRequest',
+            'X-CSRF-Token': this.token()
+          },
+          credentials: 'same-origin',
+          redirect: 'manual'
+        })
+          .then((response) => {
+            if (response.ok) {
+              // eslint-disable-next-line vue/no-mutating-props
+              this.companies = this.companies.filter((v) => v.id !== company.id)
+              this.toast('企業を削除しました。')
+            } else {
+              alert('削除に失敗しました')
+            }
+          })
+          .catch((error) => {
+            console.warn('Failed to parsing', error)
```

Comment:
```suggestion
            console.warn(error)
```

今別のIssueでアプリ全体にあるこの行をこのように修正するIssueを他の方に担当してもらっております。
ですのでこちらもこのように修正いただければありがたいです〜。